xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -1045,7 +1123,7 @@ Writes the <xref:System.Decimal> using the default <xref:System.Buffers.Standard
         <remarks>
           <format><![CDATA[  
 
-Writes the <xref:System.Double> using the default <xref:System.Buffers.StandardFormat> (i.e. 'G'). The property name is escaped before writing.
+Writes the <xref:System.Double> using the default <xref:System.Buffers.StandardFormat> (that is, 'G'). The property name is escaped before writing.
```

Comment:
`The property name is escaped before writing.` should be on its own line since its a separate remark.

```suggestion
Writes the <xref:System.Double> using the default <xref:System.Buffers.StandardFormat> (that is, 'G').

The property name is escaped before writing.
```